website/content/commands/snapshot/agent.mdx

```diff
@@ -396,9 +396,15 @@ These options cannot be used when using `backup_destinations` in a config file.
 - `-azure-blob-container-name` - Container to use. Required for Azure blob storage, and setting this
   disables local storage.
 
-* `-azure-blob-environment` - Environment to use. Defaults to AZUREPUBLICCLOUD. Other valid environments
+- `-azure-blob-environment` - Environment to use. Defaults to AZUREPUBLICCLOUD. Other valid environments
   are AZURECHINACLOUD, AZUREGERMANCLOUD and AZUREUSGOVERNMENTCLOUD. Introduced in Consul 1.7.3.
 
+~> These below options `azure-blob-service-principal-id`, `azure-blob-service-principal-secret`, and `azure-blob-tenant-id` are introduced in v1.20.1+ent.
+
+- `-azure-blob-service-principal-id` and `-azure-blob-service-principal-secret` - Use these parameters to authenticate using Service Principal (SPN) ID + Secret together.
+
+- `-azure-blob-tenant-id` - Required when using Service Principal ID and Secret.
```

Comment:
```suggestion
- `-azure-blob-tenant-id` - The ID of the tenant that owns the Azure blob.
```
It should be reworded to something similar that clearly describes the value that is required by this field.